platforms/android/library/src/androidTest/java/io/element/android/wysiwyg/test/EditorEditTextInputTests.kt

```diff
@@ -250,7 +250,7 @@ class EditorEditTextInputTests {
             .check(matches(TextViewMatcher {
                 it.editableText.getSpans<LinkSpan>().isNotEmpty()
             }))
-            .check(matches(withText("a Element to set")))
+            .check(matches(withText("a Element  to set")))
```

Comment:
If there is an explicit selection would we still want an extra space to be added?

I think it there are slightly different use cases between inserting a link at the cursor vs highlighting text and replacing with a new link and might want to check this with UX

Reply:
Hum, I suppose it could make sense in all cases where we remove something to not add the space. See my note in the initial comment as well, there are also cases where caret is at: `text | text` where we probably don't want to insert that space.

Reply:
Yeah I'm not sure what the right thing is here - I think worth asking @callumu on it and seeing what other editors are doing

Reply:
Cool, we do want that for Pills though, right ? @callumu

Reply:
Yes correct:
Pills - yes
Links - no